ci/scripts/common/build_example.sh

```diff
@@ -142,7 +150,8 @@ function build_example() {
         -DNRF5_SOFTDEVICE_VARIANT="$sd_variant" \
         -DNRF5_LINKER_SCRIPT="$(adapt_cmake_path $linker_file)" \
         -DNRF5_SDKCONFIG_PATH="$(adapt_cmake_path $sd_variant_dir/config)" \
-        -G "$build_type" || {
+        -G "$build_type" \
+        --log-level="$log_level" || {
```

Comment:
> The cmake(1) --loglevel command line option has been renamed to --log-level to make it consistent with the naming of other command line options. The --loglevel option is still supported to preserve backward compatibility.

Maybe it's better to use old version? Some installations still have CMake less than 3.16.

Reply:
Sure.